ast/builtins.go

```diff
@@ -1021,7 +1021,7 @@ var Concat = &Builtin{
 
 var FormatInt = &Builtin{
 	Name:        "format_int",
-	Description: "Returns the string representation of the number in the given base after converting it to an integer value.",
+	Description: "Returns the string representation of the number in the given base after rounding it down to an integer value.",
```

Comment:
@srenatus I smuggled this change in here, since this surprised me yesterday. Not worth a PR of its own.

Reply:
Yeah but maybe an extra commit? We can rebase and merge this PR.

Reply:
I made the change in a separate commit now. Let me know if it looks alright.